modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -75,6 +75,10 @@ type ActiveHealthChecks struct {
 	// The URI (path and query) to use for health checks
 	URI string `json:"uri,omitempty"`
 
+	// The host to use (if different from the upstream's dial
+	// address) for health checks.
+	Upstream string `json:"upstream,omitempty"`
```

Comment:
These comments show up in the JSON docs. I think this needs to be elaborated on to explain the specific usecase intended for this

Reply:
Thanks, updated but I may have gone overboard here.